assets/pages/new-contributions-landing/components/ContributionUsTicker.jsx

```diff
@@ -0,0 +1,127 @@
+// @flow
+
+// ----- Imports ----- //
+import React, { Component } from 'react';
+
+// ---- Types ----- //
+type StateTypes = {|
+  totalSoFar: number,
+  goal: number,
+|}
+
+type PropTypes = {}
+
+
+// ---- Helpers ----- //
+
+const getInitialTickerValues = () =>
```

Comment:
You may be able to add an explicit return type to this:

```jsx
Promise<{| totalSoFar: number, goal: number |}>
```

Reply:
Good idea, added.